tests/__snapshots__/ExportOrderActionTest__it_exports_international_orders_with_data_set_ups__data_set_order_mode__1.json

```diff
@@ -1,48 +1,49 @@
 {
-    "data": {
-        "shipments": [
+  "data": {
+    "shipments": [
+      {
+        "carrier": 8,
+        "customs_declaration": {
+          "contents": 1,
+          "invoice": "PDK-1",
+          "items": [
             {
-                "carrier": 8,
-                "customs_declaration": {
-                    "contents": 1,
-                    "invoice": "PDK-1",
-                    "items": [
-                        {
-                            "amount": 1,
-                            "classification": "000000",
-                            "country": "NL",
-                            "description": "test",
-                            "item_value": {
-                                "amount": 1000,
-                                "currency": "EUR"
-                            },
-                            "weight": 0
-                        }
-                    ],
-                    "weight": 1
-                },
-                "general_settings": {
-                    "save_recipient_address": 0
-                },
-                "options": {
-                    "package_type": 1,
-                    "delivery_type": 2
-                },
-                "physical_properties": {
-                    "weight": 1
-                },
-                "recipient": {
-                    "cc": "US",
-                    "city": "Cupertino",
-                    "company": "MyParcel",
-                    "person": "Felicia Parcel",
-                    "postal_code": "95014",
-                    "state": "CA",
-                    "street": "1 Infinite Loop",
-                    "vat_number": "NL123456789B01"
-                },
-                "reference_identifier": "PDK-1"
+              "amount": 1,
+              "classification": "000000",
+              "country": "NL",
+              "description": "test",
+              "item_value": {
+                "amount": 1000,
+                "currency": "EUR"
+              },
+              "weight": 0
             }
-        ]
-    }
+          ],
+          "weight": 1
+        },
+        "general_settings": {
+          "save_recipient_address": 0
+        },
+        "options": {
+          "package_type": 1,
+          "delivery_type": 2
+        },
+        "physical_properties": {
+          "weight": 1
+        },
+        "recipient": {
+          "cc": "US",
+          "city": "Cupertino",
+          "company": "MyParcel",
+          "number": "31",
```

Comment:
Hier glipt ook die 31 ertussendoor, ik weet niet of dit een indicatie is van een potentiële bug of dat die 31 ergens in de test verkeerd toegevoegd wordt? Oftewel: sanity check.